test/e2e/testdata/basic-auth.yaml

```diff
@@ -59,7 +59,6 @@ spec:
     group: gateway.networking.k8s.io
     kind: HTTPRoute
     name: http-with-basic-auth-1
-    namespace: gateway-conformance-infra
```

Comment:
what's happening here ?

Reply:
Guess I touched the `delete` key accidentally :-)